buildenv/jenkins/variables/defaults.yml

```diff
@@ -131,14 +130,24 @@ freemarker:
 openjdk_reference_repo:
   openjdk_reference_repo: '/home/jenkins/openjdk_cache'
 #========================================#
+# Cuda
+#========================================#
+cuda:
+  extra_configure_options: '--enable-cuda'
+#========================================#
+# Cuda 9
+#========================================#
+cuda_9:
```

Comment:
Should this maybe be version and not v9 specifically?

Reply:
Actually why not just have this all under the same section?

Reply:
Could. If version changed it would be less updating. If we end up with more than 1 version it would be harder to identify. But I doubt we'd use more than 1 version. I'll update.

Reply:
Windows uses cuda but with a different location. p/x linux share the same location.